tt_metal/fabric/control_plane.cpp

```
@@ -265,7 +265,7 @@ std::vector<chip_id_t> ControlPlane::get_mesh_physical_chip_ids(
                     paths.at(connected_chip_id)[paths.at(connected_chip_id).size() - 1].push_back(connected_chip_id);
                 }
             } else {
-                log_debug(
```

Might want to keep this as debug to keep the logs clean. If a mismatched graph descriptor is wrong then it should throw an error

---

ah yeah this was just me doing this for debug